src/styles/index.scss

```diff
@@ -1,3 +1,116 @@
-body {
+@import 'utils/variables';
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap';
+@import 'stars';
+
+* {
+  box-sizing: border-box;
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  &__img {
+    background-image: url(../images/imac.jpeg);
+    width: 160px;
+    height: 134px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 40px 0 4px;
+    color: $main-accent;
+  }
+
+  &__text {
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
Default value, so you can just remove it everywhere
```suggestion
```

Reply:
Done